README.md

```diff
@@ -1,8 +1,106 @@
-# Wishlist [WIP]
+# VTEX Wishlist app
 
-VTEX store lists app
+## Description
 
+The Wishlist is a canonical app that any VTEX store can use. This app is responsible for managing the costumer's lists of products.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request.
+
+## Release schedule
+
+| Release |       Status        | Initial Release | Maintenance LTS Start | End-of-life | Store Compatibility |
+| :-----: | :-----------------: | :-------------: | :-------------------: | :---------: | :-----------------: |
+|  [0.x]  | **Current Release** |   2019-04-03    |                       |             |         2.x         |
+
+See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for more information.
+
+## Table of Contents
+
+- [Usage](#usage)
+  - [Styles API](#styles-api)
+    - [CSS namespaces](#css-namespaces)
+- [Troubleshooting](#troubleshooting)
+- [Contributing](#contributing)
+- [Tests](#tests)
+
+## Usage
+
+To use this component, it's just necessary to install it in your store, using the following command:
+
+`vtex install vtex.wishlist`
+
+After that, the `ProductSummary` and the `ProductDetails` will have the button to add the product to the lists.
+
+
+#### Configuration
+
+This app has no configuration yet.
```

Comment:
Explain how the user can add wishlist link in the store through the menu builder.